specification/inference/_types/CommonTypes.ts

```diff
@@ -797,9 +829,7 @@ export class CustomRequestParams {
    * The body structure of the request. It requires passing in the string-escaped result of the JSON format HTTP request body.
    * For example:
    * ```
-   * "request":{
-   *   "content":"{\"input\":${input}}"
-   * }
+   * "request": "{\"input\":${input}}"
    * ```
    * > info
    * > The content string needs to be a single line except using the Kibana console.
```

Comment:
Should this read `except when/if using the Kibana console.`?

Reply:
Great catch, thanks!